### ISSUE TITLE
(node:7284) Warning: Deprecated environment variable: SFDX_JSON_TO_STDOUT. Please use SF_JSON_TO_STDOUT instead. (Use `node --trace-warnings ...` to show where the warning was created)     TypeError: this.config.getPluginsList is not a function

### ISSUE THREAD

**Issue by NagarjunaParala** [forcedotcom/salesforcedx-vscode#5076]
(node:7284) Warning: Deprecated environment variable: SFDX_JSON_TO_STDOUT. Please use SF_JSON_TO_STDOUT instead.
(Use `node --trace-warnings ...` to show where the warning was created)
    TypeError: this.config.getPluginsList is not a function

**Comment by CristiCanizales** [comment]
Hi @NagarjunaParala! Can you be a bit more specific on what you were trying to do, what steps were you following (to try to repro), and all the details that are in the template for creating a new issue? thanks in advance

**Comment by vleandrosf** [comment]
I'm seeing something similar.  While I have sf installed, VS Code is still only offering me SFDX commands within the Command Palette.

Log into VS Code. 
Click Extensions.  Confirm everything is updated.
Go to the Command Palette
All I see are SFDX comments (such as SFDX: Create Project).
I create a project.
Then I go to SFDX: Authorize an Org and see this:

16:55:38.914 sfdx force:auth:web:login --setalias myTest --instanceurl https://login.salesforce.com --setdefaultusername
(node:17588) Warning: Deprecated environment variable: SFDX_JSON_TO_STDOUT. Please use SF_JSON_TO_STDOUT instead.
(Use `node --trace-warnings ...` to show where the warning was created)
Warning: The "force auth web login" command has been deprecated. Use "org login web" instead.
Warning: The "--instanceurl" flag has been deprecated. Use "--instance-url | -r" instead.
Warning: The "--setdefaultusername" flag has been deprecated. Use "--set-default | -s" instead.
Warning: The "--setalias" flag has been deprecated. Use "--alias | -a" instead.

**Comment by CristiCanizales** [comment]
Hi @vleandrosf! Yes, we won't migrate to sf yet but we have some PR's open to get rid of those warnings, it's just that we'll do the merge/release only after Dreamforce.

Regarding @NagarjunaParala 's issue, I'm just curious on what he did to get that error (TypeError: this.config.getPluginsList is not a function)

**Comment by NagarjunaParala** [comment]
Hi @CristiCanizales  & @vleandrosf ,
Thank you for your kind response.
I created a SF project & trying to authorize against my Developer Sandbox. I'm facing this issue from past week. Earlier I don't see any issue.
I hope this helps.
Thanks & Regards
Nagarjuna

**Comment by CristiCanizales** [comment]
Can you try logging out of all authorized orgs? (Open the command palette and select `SFDX: Log Out from All Authorized Orgs`.) Also delete all your expired scratch orgs, if you have any. (Open the command palette and select `SFDX: Remove Deleted and Expired Orgs`). Then, reload VS Code and try authorizing again. Let me know if that helps!
